internal/internal_workflow.go

```diff
@@ -260,6 +262,7 @@ var _ WaitGroup = (*waitGroupImpl)(nil)
 var _ dispatcher = (*dispatcherImpl)(nil)
 
 var stackBuf [100000]byte
+var debugMode = os.Getenv("TEMPORAL_DEBUG_MODE") == "true"
```

Comment:
Nit: The java side used parseBoolean which I think will handle odd capitalizations and some other truthy values. For flags like this I like the idiom of if they're set to anything, they're true, and false is the absence of them. But, maybe better to be consistent with Java, and just lowercase the string or something before comparing.

Reply:
Yeah, I would check for non-empty value and treat it as `true`. Also I would name it `TEMPORAL_DEBUG`, but this is up to you.

Reply:
Both are good comments. I was thinking about doing an empty check but decided to do it this way for consistency reasons with java implementation, and in java it's that way also for consistency reasons with other flags. But I totally agree that doing an empty check would be more idiomatic. Let me change it to that in both go and java.